app/models/event.rb

```diff
@@ -7,4 +9,9 @@ class Event < ApplicationRecord
   validates :capacity, presence: true, numericality: { greater_than: 0 }
 
   scope :active, -> { where cancelled: false }
+  scope :upcoming, -> { where 'time > ?', DateTime.now }
```

Comment:
Use `Time.current`. Although very similar to `DateTime.now` but a standard one used a cross rails apps.